Development

Sheyne opened the pull request.

danielscottjames commented:
I checked out this branch and built it and ran `npm pack`, and I'm getting:
```
ERROR in /Users/Daniel/Development/sinap-ide/node_modules/sinap-core/lib/src/plugin.d.ts
    (1,1): error TS6053: File '/Users/Daniel/Development/sinap-ide/node_modules/sinap-core/typings/modules/uuid/index.d.ts' not found.
```

Sheyne commented:
@RighteousRaichu try a typings install

Sheyne commented:
This isn't ready until we have a plan for implementing graph controller.

Sheyne committed:
feat(Plugin): has access to it's directory

by Dyllon

Sheyne committed:
use sinap-types

Sheyne committed:
comments

Sheyne committed:
fix package.json

Sheyne committed:
progress

Sheyne committed:
restructuring

Sheyne committed:
bugfix

Sheyne committed:
version bump

Sheyne committed:
Update README.md

Sheyne committed:
Update README.md

Sheyne committed:
Update README.md

Sheyne committed:
add a simple test

Sheyne committed:
bump version number

Sheyne committed:
Initialize models

Sheyne committed:
add a validate edge method

Sheyne committed:
alias Element

Sheyne committed:
first crack at serialization

Sheyne committed:
Stupidity error

Sheyne committed:
stupidity error

Sheyne committed:
support for unwrapping Values

This means turning them into native JS
with full support for cycles and whatnot

Sheyne committed:
move unvalue to sinap-typescript

Sheyne committed:
bugfix

Sheyne committed:
Update plugin.ts

Sheyne committed:
dyllon's changes

slaymaker1907 committed:
Fixed tests to actually do something.

Also added tests for getInterprterInfo.

slaymaker1907 committed:
Added a clean command.

Sheyne committed:
setup cyclic dep
